main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1557,7 +1571,7 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
     //
     // Check if the definition is marked public.
     //
-    if (sig0.mod.isPublic) // MATT check instance availability instead?
+    if (sig0.spec.mod.isPublic) // MATT check instance availability instead?
```

Comment:
What is this comment?

Reply:
From when we were considering how to handle visibility, including private instances, etc. I think we should actually be looking at class availablity, since signatures now have to be public. I'll revisit outside this PR.